recipes/repo.rb

```diff
@@ -265,6 +265,162 @@
   end
 end
 
+# Pulp setup
+pulp_data_directory = '/var/repos/.pulp'
+user 'pulp' do
+  manage_home true
+  comment 'Pulp content manager'
+  uid '1200'
+end
+group 'docker' do
+  append true
+  members ['pulp']
+  action [:manage]
+end
+directory pulp_data_directory do
+  owner 'pulp'
+  mode '0700'
+end
+directory "#{pulp_data_directory}/media" do
+  owner 'pulp'
+end
+
+cookbook_file "#{pulp_data_directory}/initialize.py" do
+  owner 'pulp'
+  source 'rpm_repo_init.py'
+  mode '0600'
```

Comment:
If we're going to execute the script explicitly with the interpreter, we should drop the shebang from `rpm_repo-init.py`.

Reply:
Do you have a preference for executable vs otherwise? I don't.

Reply:
In the spirit of deterministic behavior, I'd guess my preference would be to invoke it using the interpreter directly, and not to rely on the shebang. But I don't really care very much either, just looking for consistency.